externs/v20140218_eme.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2014 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @fileoverview Externs for prefixed EME v20140218
+ * (http://www.w3.org/TR/2014/WD-encrypted-media-20140218).
+ * @externs
+ */
+
+
+
+/**
+ * @constructor
+ * @param {string} keySystem
+ */
+function MSMediaKeys(keySystem) {}
+
+
+/**
+ * @param {string} keySystem
+ * @param {string} contentType
+ * @return {boolean}
+ */
+MSMediaKeys.isTypeSupported = function(keySystem, contentType) {};
+
+
+/**
+ * @param {string} contentType
+ * @param {!ArrayBuffer|ArrayBufferView|null} initData
+ * @return {MSMediaKeySession}
+ */
+MSMediaKeys.prototype.createSession = function(contentType, initData) {};
+
+
+
+/**
+ * @interface
+ */
+function MSMediaKeySession() {}
+
+
+/**
+ * @param {ArrayBuffer|ArrayBufferView|null} message
+ */
+MSMediaKeySession.prototype.update = function(message) {};
+
+MSMediaKeySession.prototype.close = function() {};
+
+
+/**
+ * @param {string} eventType
+ * @param {Function} callback
+ */
+MSMediaKeySession.prototype.addEventListener = function(eventType, callback) {};
```

Comment:
When MSMediaKeySession extends EventTarget, this can simply be annotated override.  See mediakeys.js for an example.